packages/turbo-repository/rust/src/lib.rs

```diff
@@ -35,18 +42,46 @@ pub struct Workspace {
     /// The package manager used by the workspace.
     #[napi(readonly)]
     pub package_manager: PackageManager,
+    /// The package graph for the workspace.
+    graph: PackageGraph,
 }
 
+#[napi]
 impl Package {
-    fn new(workspace_path: &AbsoluteSystemPath, package_path: &AbsoluteSystemPath) -> Self {
+    fn new(
+        name: String,
+        workspace_path: &AbsoluteSystemPath,
+        package_path: &AbsoluteSystemPath,
+    ) -> Self {
         let relative_path = workspace_path
             .anchor(package_path)
             .expect("Package path is within the workspace");
         Self {
+            name: name,
```

Comment:
Shorthand works in rust too!

```suggestion
            name,
```